crates/bevy_asset/src/asset_server.rs

```diff
@@ -193,6 +216,7 @@ impl AssetServer {
         }
     }
 
+    /// Gets the overall load state of a group of assets from the provided handles.
```

Comment:
We should clarify that the group will only be `Loaded` if all assets are loaded.